ocaml/xapi/xapi_globs.ml

```diff
@@ -972,6 +972,8 @@ let pvsproxy_close_cache_vdi = ref "/opt/citrix/pvsproxy/close-cache-vdi.sh"
 
 let yum_repos_config_dir = ref "/etc/yum.repos.d"
 
+let dnf_repo_config_file = ref "/etc/dnf/repos.override.d/99-config_manager.repo"
```

Comment:
Why is this a ref when there is no code to modify it?

Reply:
Just do same as others e.g. `/etc/yum.repos.d`